src/identity/IdentityProviderHttpHandler.ts

```diff
@@ -69,6 +69,12 @@ export class IdentityProviderHttpHandler extends HttpHandler {
       await this.interactionHttpHandler.canHandle({ ...input, provider });
     } catch {
       this.logger.debug(`Sending request to oidc-provider: ${input.request.url}`);
+      provider.use(async(ctx, next): Promise<void> => {
+        // eslint-disable-next-line callback-return
```

Comment:
Also, I think ESlint is right here. We should not do things after return.
Hope @joachimvh's suggestion works.